clients/go/gengo.go

```diff
@@ -69,6 +69,12 @@ type WagClient struct {
 
 var _ Client = (*WagClient)(nil)
 
+// TODO this is a starting point
+var defaultTransport = &http.Transport{
+    MaxIdleConns:          512,
```

Comment:
This seems high... I'd expect most services to max out at roughly their max number of concurrent requests

Reply:
I did `2^ones digit of second` 🙈